### DISCUSSION TITLE
🐛 [Bug]:  cdn地址管了， 替代cdn

### DISCUSSION THREAD

**Discussion by qian-chen97** [opentiny/tiny-engine#422] [Q&A]
### Environment

windwos11

### Version

16

### Version

最新

### Link to minimal reproduction

1

### Step to reproduce

1

### What is your project name

1

[a comment by an automated account is not shown]

**Comment by jxhhdx** [comment]
+1现在要选择哪一个cdn比较好

[a comment by an automated account is not shown]

**Comment by chilingling** [comment]
请描述清楚场景，需求。
@ArtisanQ @jxhhdx

**Reply by jxhhdx** [reply]
<img width="1009" alt="image" src="https://github.com/opentiny/tiny-engine/assets/40119767/e67b2ab0-bf3e-4227-81f9-d0b8ba47ae40">
前段时间https://cdn.onmicrosoft.cn突然无法获取资源了，然后大家这个域名说自己停止维护，所以说tiny-engine要不要把这个域名换掉。现在能用是能用，但是不知道之后会不会有问题

**Reply by chilingling** [reply]
https://github.com/cnpm/cnpmcore/issues/674

在尝试让 npmmirror 支持在 import map 场景下使用的 unpkg 能力，支持后替换成 npmmirror。
